source-mongodb/change_stream.go

```diff
@@ -137,7 +137,7 @@ func (c *capture) initializeStreams(
 			return nil, fmt.Errorf("initializing change stream on database %q: %w", db, err)
 		}
 
-		logEntry.Info("intialized change stream")
+		logEntry.Debug("intialized change stream")
```

Comment:
Same comment here

Reply:
In practice I found these logs to be quite redundant with the "catching up streams" info logging. We always initialize a change stream for every database when this happens so its not super useful.